express-api/src/utilities/authorizationChecks.ts

```diff
@@ -56,3 +56,16 @@ export const isUserActive = async (kcUser: KeycloakUser): Promise<boolean> => {
   const user = await getUser(kcUser.preferred_username);
   return user.Status === 'Active';
 };
+
+export const checkUserPermission = async (
```

Comment:
Just thinking we should be more specific with this name maybe. It really only checks if the user has permissions to view something. 
If this was any other operation, we wouldn't want to give the auditor role the option to change something.
```suggestion
export const checkReadPermission = async (
```